frontend/src/core/app/permissions.js

```diff
@@ -72,6 +72,48 @@ define(function(require) {
 			
 		},
 
+		hasSuperPermissions:function(){
+			var sessionModelPermissions = Origin.sessionModel.get('permissions');
+			var hasWildCard = false;
+
+			_.each(sessionModelPermissions, function(permission) {
```

Comment:
By using  `['{{tenantid}}/*:delete']` and `['*/*:delete']`  not able to differentiate between tenant Admin and super Admin, as  `the permissions.hasPermissions`  returns true for both